kits/rust/README.md

```diff
@@ -0,0 +1,97 @@
+# Lux AI Season 2 Rust Kit
+
+This is a rust implementation inteded for enabling communication and validation
+of agent actions, as well as providing some helpful utilities for interacting with
+game rules.
+
+**Disclaimer**: Although this kit is heavily documented and  reasonably tested.
+It has been implemented by a third party based on the specification and python
+implementation. As such, any deviation from the python implementation with
+regards to documentation or behaviour is unintended and should be logged as a bug
+
+## Implementation Clarifications
+
+- What's the use of factions / should names outside of the 4 valid values be supported
```

Comment:
Factions are currently unused, but there is only those 4 values. Feel free to remove this section.